### PR TITLE
Support loading build history.

### PR DESCRIPTION
**Pull request by TamalSaha**

### CODE REVIEW

File: job.go [hunk 1]
```diff
@@ -459,3 +466,18 @@ func (j *Job) Poll() (int, error) {
...
 	return response.StatusCode, nil
 }
+
+func (j *Job) History() ([]*history, error) {
+	endpoint := j.Raw.URL[strings.Index(j.Raw.URL, "/job"):]
+	if !strings.HasSuffix(endpoint, "/") {
+		endpoint += "/"
+	}
+	endpoint += "buildHistory/ajax"
+	fmt.Println(endpoint)
```
Comment (bndr) [inline job.go]: This print should not be here

**Comment by bndr** [comment]
Some tests would be nice, to test the new endpoint.

**Comment by TamalSaha** [comment]
@bndr I have added tests. Tests pass on my local Jenkins. They seem to get stuck on Travis. I am not really familiar with Travis. Do you mind taking a look?

**Comment by bndr** [comment]
Yeah sure, thanks for the new feature!

**Comment by TamalSaha** [comment]
@bndr tests are passing now. Please take a look.

**Comment by bndr** [comment]
Thanks again, sorry I didnt have time to do the tests myself.

**Pull request merged**